navigation.py

```diff
@@ -0,0 +1,237 @@
+import time
+from selenium import webdriver
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import Select
+
+from custom_exceptions import *
+
+pages = {}
+pages["root"] = "http://kbhff.local/"
+pages["root"] = "http://pre-launch.kbhff.dk/"
+pages["login"] = pages["root"] + "login"
+pages["signup"] = pages["root"] + "bliv-medlem"
+
+def navigate_to_page(page_name, driver):
+    """Navigate to the specified page.
+
+    Positional arguments:
+        page_name -- a string corresponding to the desired page. Supported values are:
+            login
+            bekraeft_konto
+            opret_adgangskode
+            bekraeft_konto_receipt: /bliv-medlem/bekraeft/kvittering
+            signup
+        driver -- the Selenium driver to use
+    page names not yet implementer:
```

Comment:
"implemented"

Reply:
And given that we explicitly check that the page passed is implemented, and provide a (dynamically updated) list of supported pages, I would remove this comment. I think we're pretty likely to forget to keep it updated...